src/commands/sns_summary.rs

```diff
@@ -0,0 +1,31 @@
+use crate::commands::sns_canisters_summary::SnsCanistersSummaryOps;
+use crate::commands::{nervous_system_parameters, sns_canisters_summary};
+use crate::lib::signing::IngressWithRequestId;
+use crate::lib::AnyhowResult;
+use crate::SnsCanisterIds;
+use clap::Parser;
+
+#[derive(Parser, Debug)]
+#[clap(about, version, author)]
+pub struct SnsSummaryOps {
```

Comment:
Please add a `///` style comment at the top of this method (you can copy the one you write in src/commands/mod.rs. Each command written so far has kept with this convention and I think we should continue that.